stats.go

```diff
@@ -100,6 +116,17 @@ func scenarioWorker(scen *Scenario, sortedTimesPlayed *[]*Scenario, uniqueDays *
 	scen.ChartByDate = ScenarioLineChart(scen)
 }
 
+func calculateWMA(scen *Scenario) {
+	wma := NewWMA(DefaultWMAWindow)
+	for _, dateScores := range scen.ByDateScores {
+		for date, scores := range dateScores {
+			wma.Add(scores...)
+			avg, count := wma.Average()
+			scen.ByDateWMA = append(scen.ByDateWMA, map[string]DateWMA{date: {Avg: float64(int(avg*10)) / 10, Grouped: count}})
```

Comment:
I feel like this line could be shorter and do less.
How would you feel about moving the following to a separate line?
`DateWMA{date: {Avg: float64(int(avg*10)) / 10, Grouped: count}}`

Resulting in:
```
dateWMA := DateWMA{Avg: float64(int(avg*10)) / 10, Grouped: count}
scen.ByDateWMA = append(scen.ByDateWMA, map[string]DateWMA{date: dateWMA})
```

Reply:
I've made this change but my PR has broken the unit test in stats_test.go. I'll fix that shortly.

Reply:
makes sense, thank you!